### DISCUSSION TITLE
Failed setup - Modbus error

### DISCUSSION THREAD

**Discussion by hibberts** [WillCodeForCats/solaredge-modbus-multi#614] [General]
WillCodeForCats, thanks for this. I’m new to the HA platform and got it just so that I could control my SE system. It’s taken bit of learning, but I’m getting there. I’ve installed the program but get the following warning
![SE warning](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/173318043/2b4ce078-1b89-49d6-bf6d-f7b62ece231c)
![SE](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/173318043/d7700a04-55a6-45bb-b6d6-ce8e6e24856a)
![SE Config](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/173318043/d30fa2c8-084c-4e8c-b0e3-d046ad882c55)

If there is anything that you could recommend that would be greatly appreciated
Thanks RH

**Comment by rjkbj** [comment]
I am seeing the same error, but mine uses port 1502.  Did you ever figure this one out?
